### DISCUSSION TITLE
Change position of the £ sign in Yetiforce

### DISCUSSION THREAD

**Discussion by MonsterITServices** [YetiForceCompany/YetiForceCRM#16793] [How to] [answered]
Hi, How can I change the £ symbol from the end to make it at the beginning?

![image](https://user-images.githubusercontent.com/22075161/210549563-cd814f39-71e4-464d-a197-34e6dadb4958.png)

Thank you

Alex

**Comment by apcloic** [comment, marked as answer]
@alextovey
In user preferences.
https://gitstable.yetiforce.com/index.php?module=Users&view=PreferenceDetail&record=5
